pkg/minikube/node/start.go

```diff
@@ -191,6 +191,23 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		go addons.Start(&wg, starter.Cfg, starter.ExistingAddons, addonList)
 	}
 
+	// discourage use of the virtualbox driver
+	if starter.Cfg.Driver == driver.VirtualBox && viper.GetBool(config.WantVirtualBoxDriverWarning) {
+		var altDriverList strings.Builder
+		for _, d := range driver.Choices(true) {
+			if d.Name != "virtualbox" {
+				altDriverList.WriteString(fmt.Sprintf("\t- %s\n", d.Name))
+			}
+		}
+
+		out.ErrT(style.Empty, "")
```

Comment:
instead of this Err, lets use this helper

out.Boxed(`There are alternative drivers to virtualbox for better performance, support consider using altDriverList.String()) . to turn this warning off use `minikube config set ...'`)